app/activeplan/homepage/index.njk

```diff
@@ -0,0 +1,137 @@
+{% extends "common/templates/layout.njk" %}
+{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/tabs/macro.njk" import govukTabs %}
+{% from "moj/components/button-menu/macro.njk" import mojButtonMenu %}
+
+{% set heading = "Active plan" %}
+{% set errorHeading = "Error - " + heading %}
+{% set addObjectiveUrl = "/individual-id/" + id + "/edit-plan/" + planId + "/edit-objective/NEW" %}
+{% set planRoute = "/individual-id/" + id + "/plan/" + planId %}
+{% set printFullPlanUrl = planRoute + "/print-full" %}
+{% set printSimplePlanUrl = planRoute + "/print-simple" %}
+{% set endPlanUrl = planRoute + "/end-plan" %}
+{% set addMeetingUrl = planRoute + "/sentence-board-review" %}
+{% set contactUrl = planRoute + "/contact-arrangements" %}
+
+{% block pageTitle %}
+    {% if errors %}
+        {{ errorHeading }}
+    {% else %}
+        {{ heading }}
+    {% endif %}
+{% endblock %}
+
+{% block beforeContent %}
+
+{% endblock %}
+
+{% block content %}
+
+    {# include errorSummary partial #}
+    {{ super() }}
+    {% if planStarted %}
+        <div class="moj-banner moj-banner--success">
+            <svg class="moj-banner__icon" fill="currentColor" role="presentation" focusable="false"
+                 xmlns="http://www.w3.org/2000/svg" viewBox="0 0 25 25" height="25" width="25">
+                <path d="M25,6.2L8.7,23.2L0,14.1l4-4.2l4.7,4.9L21,2L25,6.2z"></path>
+            </svg>
+            <div class="moj-banner__message"><span class="moj-banner__assistive">Success</span>The plan has been started
+            </div>
+        </div>
+    {% endif %}
+    <div class="moj-page-header-actions">
+        <div class="moj-page-header-actions__title">
+            <h2 class="govuk-heading-l">
+                {{ heading }}
+            </h2>
+        </div>
+
+        <div class="moj-page-header-actions__actions moj-button-menu__float-right">
+            {{ mojButtonMenu({
+                classes: 'active-plan-menu',
+                items: [{
+                    text: 'Add an objective',
+                    href: addObjectiveUrl,
+                    classes: 'govuk-button--secondary moj-button-menu__item moj-page-header-actions__action'
```

Comment:
Probably pedantic, but it does make the nunjucks component easier to read: 
This class set is used three times, could just:
{% set menuButtonClasses='govuk-button--secondary moj-button-menu__item moj-page-header-actions__action' %}
or catch the last one too?
{% set buttonClasses='moj-button-menu__item moj-page-header-actions__action' %}
{% set secondaryButtonClasses='govuk-button--secondary'+buttonClasses %}